scripts/build/Jenkins/Jenkinsfile

```diff
@@ -415,6 +415,18 @@ def ExportTestResults(Map options, String platform, String type, String workspac
     }
 }
 
+def ExportTestScreenshots(Map options, String workspace, String platformName, String jobName, Map params) {
+    catchError(message: "Error exporting test screenshots (this won't fail the build)", buildResult: 'SUCCESS', stageResult: 'FAILURE') {
+        def screenshotsFolder = '${workspace}/${ENGINE_REPOSITORY_NAME}/AutomatedTesting/user/PythonTests/Automated/Screenshots'
+        def s3Uploader = '${workspace}/${ENGINE_REPOSITORY_NAME}/scripts/build/tools/upload_to_s3.py'
+        def command = '${options.PYTHON_DIR}/python.cmd -u ${s3Uploader} --base_dir ${screenshotsFolder} ' +
```

Comment:
Probably want to add a `--extra-args {'ACL': 'bucket-owner-full-control'}` as an additional argument. This allows cross account uploads to work correctly (which occurs if we move between sandbox and AR environments)